src/utils/components/FileInput.vue

```diff
@@ -265,7 +267,10 @@ function onFileSelected(evt) {
                 data-cy="file-input-invalid-feedback"
             >
                 {{
-                    i18n.t(invalidMessage).replace('ALLOWED_FORMATS', acceptedFileTypes.join(', '))
+                    i18n
+                        .t(invalidMessage)
+                        .replace('ALLOWED_FORMATS', acceptedFileTypes.join(', '))
+                        .replace('MAX_FILE_SIZE', maxFileSizeHuman)
```

Comment:
might make sense to use some of our I18N tool here instead
https://vue-i18n.intlify.dev/guide/essentials/syntax.html#interpolations

Reply:
good idea